Source/WebKit/UIProcess/ios/fullscreen/WKFullScreenWindowControllerIOS.mm

```diff
@@ -881,7 +882,7 @@ - (void)enterFullScreen:(CGSize)mediaDimensions
     _lastKnownParentWindow = [webView window];
     _parentWindowState = adoptNS([[WKFullScreenParentWindowState alloc] initWithWindow:_lastKnownParentWindow.get()]);
 #if ENABLE(QUICKLOOK_FULLSCREEN)
-    _isImageElement = manager->isImageElement();
+    _isImageElement = manager->isImageElement() && WTF::processHasEntitlement("com.apple.surfboard.chrome-customization"_s);
```

Comment:
might be a good opportunity to rename `_isImageElement` to `_isUsingQuickLook`

Reply:
Good idea!